client.js

```diff
@@ -31,7 +31,7 @@ class TrinoClient {
     })
   }
 
-  _request(query, bodyStream, nextUri, isCancelled) {
+  _request(query, bodyStream, nextUri, isCancelled, headers) {
```

Comment:
As the number of parameters grow, consider named parapmeter using the [roro pattern](https://www.freecodecamp.org/news/elegant-patterns-in-modern-javascript-roro-be01e7669cbd/)